openwisp_network_topology/api/views.py

```diff
@@ -177,28 +177,28 @@ def get(self, request, pk, format=None):
             return Response({'detail': _('invalid date supplied')}, status=403)
 
 
-class ProtectedAPIMixin(FilterByOrganizationManaged):
-    authentication_classes = [BearerAuthentication, SessionAuthentication]
-    permission_classes = [
-        IsAuthenticated,
```

Comment:
It may be worth to try using the same permission classes as in the other modules here to see if any test breaks

Reply:
these `permission_classes` works, thanks!
```python
permission_classes = (
    IsOrganizationManager,
    DjangoModelPermissions,
)
```

Reply:
![Screenshot from 2022-12-14 14-42-33](https://user-images.githubusercontent.com/56113566/207554671-ae2993ca-c440-4de5-8a0d-b1e5a1704ddd.png)